Python/21.py

```diff
@@ -0,0 +1,10 @@
+def GCD(a,b):
+    if b==0:
+        return a
+    r=a%b
+    return GCD(b,r)
+numbers=input()
```

Comment:
@kunal817 Please be more descriptive in your programs, (Use Print statement to inform the user that He/She needs to enter 2 numbers to calculate the GCD rather than directly taking the input.)

Reply:
Thank you for your suggestions. I have made the required changes